package.json

```diff
@@ -1,62 +1,66 @@
-	"dependencies": {
-		"bonjour-service": "^1.2.1",
-		"debug": "^4.3.7",
-		"ws": "^8.18.0"
-	},
-	"devDependencies": {
-		"@ava/typescript": "^5.0.0",
-		"@types/debug": "^4.1.12",
-		"@types/node": "^22.5.5",
-		"@types/ws": "^8.5.12",
-		"@typescript-eslint/eslint-plugin": "^8.8.1",
-		"@typescript-eslint/parser": "^8.8.1",
-		"ava": "^6.1.3",
-		"copyfiles": "^2.4.1",
-		"eslint": "^8.57.1",
-		"eslint-config-prettier": "^9.1.0",
-		"eslint-config-semistandard": "^17.0.0",
-		"eslint-plugin-prettier": "^5.2.1",
-		"eslint-plugin-promise": "^6.6.0",
-		"prettier": "^3.3.3",
-		"ts-node": "^10.9.2",
-		"typescript": "5.5.x"
-	},
-	"scripts": {
-		"format": "prettier --write \"./{examples,lib,test}/**/*.ts\"",
-		"lint": "prettier --write \"./{examples,lib,test}/**/*.ts\" && eslint \"./{examples,lib,test}/**/*.ts\" --fix",
-		"test": "npm run build && ava dist/test/**/*.js",
-		"build": "tsc && copyfiles \"{examples,lib,test}/**/*.json\" dist/"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/unfoldedcircle/integration-node-library.git"
-	},
-	"keywords": [
-		"uc",
-		"unfoldedcircle",
-		"remotetwo",
-		"smartremote",
-		"api",
-		"integration"
-	],
-	"author": "Unfolded Circle",
-	"license": "ISC",
-	"bugs": {
-		"url": "https://github.com/unfoldedcircle/integration-node-library/issues"
-	},
-	"homepage": "https://unfoldedcircle.com",
-	"private": true,
-	"engines": {
-		"node": ">=16.18.0"
-	}
+  "name": "uc-integration-api",
+  "description": "NodeJS wrapper for the Unfolded Circle Integration API",
+  "version": "0.1.0",
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "directories": {
+    "example": "examples",
+    "lib": "lib"
+  },
+  "dependencies": {
+    "bonjour-service": "^1.2.1",
+    "debug": "^4.3.7",
+    "ws": "^8.18.0"
+  },
+  "devDependencies": {
+    "@ava/typescript": "^5.0.0",
+    "@types/debug": "^4.1.12",
+    "@types/node": "^22.5.5",
+    "@types/ws": "^8.5.12",
+    "@typescript-eslint/eslint-plugin": "^8.8.1",
+    "@typescript-eslint/parser": "^8.8.1",
+    "ava": "^6.1.3",
+    "copyfiles": "^2.4.1",
+    "eslint": "^8.57.1",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-semistandard": "^17.0.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-promise": "^6.6.0",
+    "prettier": "^3.3.3",
+    "ts-node": "^10.9.2",
+    "tsup": "^8.3.0",
+    "typescript": "5.5.x"
+  },
+  "scripts": {
+    "format": "prettier --write .",
+    "code-check": "prettier --check . && eslint .",
+    "lint": "eslint --fix",
+    "build": "tsup",
+    "test": "npm run build && ava dist/test/**/*.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/unfoldedcircle/integration-node-library.git"
+  },
+  "keywords": [
+    "uc",
+    "unfoldedcircle",
+    "remotetwo",
+    "smartremote",
+    "api",
+    "integration"
+  ],
+  "author": "Unfolded Circle",
+  "license": "ISC",
```

Comment:
The `license` field was not set correctly in the original project. Please set to `Apache-2.0`, then update packages to also set in package-lock.json

Reply:
I changed it and updated the packages.